back-end/sports_platform/sports_platform_api/tests/test_recover.py

```diff
@@ -0,0 +1,92 @@
+from django.test import Client, TestCase
+from ..models import User
+from rest_framework.authtoken.models import Token
+from django.contrib.auth import authenticate
+from .test_helper_functions import create_mock_user
+from django.core import mail
+
+class LoginTest(TestCase):
```

Comment:
I guess the class name should be changed

Reply:
Changed, thanks for pointing out.